src/malloc/dlmalloc/malloc.c

```diff
@@ -846,6 +846,29 @@ extern "C" {
 #define dlindependent_calloc   independent_calloc
 #define dlindependent_comalloc independent_comalloc
 #define dlbulk_free            bulk_free
+#else /* USE_DL_PREFIX */
+
+#ifdef __CHEERP__
+#define CHEERP_ASMJS_ATTRIB __attribute((cheerp_asmjs))
+#else
+#define CHEERP_ASMJS_ATTRIB 
+#endif
+
+#define CHEERP_DEFINE_MEMFUNC(name, attrib, ret, ...) \
+  attrib __attribute__ ((__weak__, alias("dl"#name))) ret name(__VA_ARGS__); \
+  attrib __attribute__ ((alias("dl"#name))) ret __builtin_cheerp_##name(__VA_ARGS__);
```

Comment:
change the name of these. __builtin_stuff usually means something that is replaced by an intrisic by clang. This is a bit different. so something like __cheerp_malloc